Services/Init/classes/class.ilInitialisation.php

```diff
@@ -1723,17 +1710,8 @@ protected static function initHTML()
 
 		if(ilContext::hasUser())
 		{
-			// LTI
-			if ($lti->isActive())
-			{
-				include_once './Services/LTI/classes/class.ilMainMenuGUI.php';
-				$ilMainMenu = new LTI\ilMainMenuGUI("_top");
-			}
-			else
-			{
-				include_once './Services/MainMenu/classes/class.ilMainMenuGUI.php';
-				$ilMainMenu = new ilMainMenuGUI("_top");
-			}
+			include_once './Services/MainMenu/classes/class.ilMainMenuGUI.php';
+			$ilMainMenu = new ilMainMenuGUI("_top");
```

Comment:
I will have a look at this after the merge, I think this now can be removed as well.